### DISCUSSION TITLE
Outreachy 2024: Accessibility review and accessibility improvement of the Ushahidi Platform Client

### DISCUSSION THREAD

**Discussion by CeciliaHinga** [ushahidi/platform#4713] [Outreachy and Google Summer of Code]
Welcome to Ushahidi and the application process for Outreachy 2024!

We are so excited you are here!!

We have compiled a list of helpful resources and steps to help you make successful Outreachy contributions. Ensure that you do not skip any of the steps below.

**Step 1** - Get familiar with the Ushahidi community:
- Join, say hello, and mention your country in our [community Discord server](https://discord.com/invite/p6BAVx7T9K). Use the correct channel (**_#outreachy-accessibility_**) to communicate, help, and get help from the community.
- Read through our Code of Conduct in our #welcome and rules channel on Discord.

**Step 2** - Get your developer environment setup (If you get stuck, reach out to the community Discord channel to get help):
- Check out the instructions to set up and launch the developer-environment here: https://docs.ushahidi.com/platform-developer-documentation/v/set-up-the-developer-environment-for-ushahidi/

**Step 3** - Share your successful setup: Once you have completed steps 1-2, send a reply to the discord channel with screenshots of your setup in the browser and select an issue to work on.

**Step 4** - Explore the Ushahidi Platform and the different concepts like surveys (also called "forms" in some places) and posts. You can explore by studying and clicking around on the platform's user interface or through our [user documentation](https://docs.ushahidi.com/platform-user-manual/) 

**Step 5** - You can find issues to work on through our [[OSS community board](https://github.com/orgs/ushahidi/projects/18/views/1?sliceBy%5Bvalue%5D=Outreachy%3A+Accessibility+2024)]. We have made it easy - check for issue tickets with the outreachy2024 label on them, you will also find the difficulty level on the issue tickets to help you easily identify those that you can tackle based on your experience and/or strengths. Take note of these when working on issues:

**_Things to Note_**

- You can only work on one issue at a time. We are looking for the quality of the contributions rather than the number of contributions when evaluating the applications.

- Once you are assigned to an issue, we want you to submit a pull request within 2 days, even if it is not yet complete or a working solution. This is to make sure our contributor's work is "in the open," as is often done in Open Source, and make sure no issues are not being worked on. If you don't show progress within 3 days, we will open it up for other participants. GitHub has a good guide on [how to submit pull requests from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).

- Remember that Ushahidi is an open-source community. The mentors are trying their best to reply to your questions, but there may be some delays. By helping other contributors out in our community Discord server, you are also helping the mentors. We recognize this and see it as something very positive when evaluating your submissions. So if you see someone asking for help in the chat and know the answer, please help them out! Keep the tone nice and collaborative.

Welcome once again, we are looking forward to the awesome contributions you will submit to our community 🌻 🎉 💐

**Comment by mildrette** [comment]
hello! i am mildred fonka an Outreachy intern for 2024. I am happen to be here and to contribute to the Ushahidi project.

**Reply by Angamanga** [reply]
Welcome @mildrette! 🎉

**Reply by mildrette** [reply]
thank you very much, I 
am really glad to be here

**Comment by Melodee618** [comment]
Hi everyone, my name is Melodee and I'm Outreachy applicant. I'm super excited to contribute to this project and learn from you all!

**Reply by Angamanga** [reply]
Welcome @Melodee618! 🎉

**Comment by rajutkarsh07** [comment]
Hello everyone, I'm Utkarsh Raj, an Outreachy applicant. I'm thrilled about the opportunity to contribute to this project and eager to learn from all of you!

**Reply by Angamanga** [reply]
Welcome @rajutkarsh07! 🎉

**Comment by codewithmide** [comment]
> Welcome to Ushahidi and the application process for Outreachy 2024!
> 
> We are so excited you are here!!
> 
> We have compiled a list of helpful resources and steps to help you make successful Outreachy contributions. Ensure that you do not skip any of the steps below.
> 
> **Step 1** - Get familiar with the Ushahidi community:
> 
> * Join, say hello, and mention your country in our [community Discord server](https://discord.com/invite/p6BAVx7T9K). Use the correct channel (**_#outreachy-accessibility_**) to communicate, help, and get help from the community.
> * Read through our Code of Conduct in our #welcome and rules channel on Discord.
> 
> **Step 2** - Get your developer environment setup (If you get stuck, reach out to the community Discord channel to get help):
> 
> * Check out the instructions to set up and launch the developer-environment here: https://docs.ushahidi.com/platform-developer-documentation/v/set-up-the-developer-environment-for-ushahidi/
> 
> **Step 3** - Share your successful setup: Once you have completed steps 1-2, send a reply to the discord channel with screenshots of your setup in the browser and select an issue to work on.
> 
> **Step 4** - Explore the Ushahidi Platform and the different concepts like surveys (also called "forms" in some places) and posts. You can explore by studying and clicking around on the platform's user interface or through our [user documentation](https://docs.ushahidi.com/platform-user-manual/)
> 
> **Step 5** - You can find issues to work on through our [[OSS community board](https://github.com/orgs/ushahidi/projects/18/views/1?sliceBy%5Bvalue%5D=Outreachy%3A+Accessibility+2024)]. We have made it easy - check for issue tickets with the outreachy2024 label on them, you will also find the difficulty level on the issue tickets to help you easily identify those that you can tackle based on your experience and/or strengths. Take note of these when working on issues:
> 
> **_Things to Note_**
> 
> * You can only work on one issue at a time. We are looking for the quality of the contributions rather than the number of contributions when evaluating the applications.
> * Once you are assigned to an issue, we want you to submit a pull request within 2 days, even if it is not yet complete or a working solution. This is to make sure our contributor's work is "in the open," as is often done in Open Source, and make sure no issues are not being worked on. If you don't show progress within 3 days, we will open it up for other participants. GitHub has a good guide on [how to submit pull requests from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
> * Remember that Ushahidi is an open-source community. The mentors are trying their best to reply to your questions, but there may be some delays. By helping other contributors out in our community Discord server, you are also helping the mentors. We recognize this and see it as something very positive when evaluating your submissions. So if you see someone asking for help in the chat and know the answer, please help them out! Keep the tone nice and collaborative.
> 
> Welcome once again, we are looking forward to the awesome contributions you will submit to our community 🌻 🎉 💐

Thank you @CeciliaHinga for the descriptive information. I look forward to making quality contributions

**Reply by Angamanga** [reply]
Welcome @codewithmide! 🎉

**Comment by Mekebib-Ergudo** [comment]
Hello everyone,

I'm Mekebib, and I'm thrilled to join this amazing team! Looking forward to working together and achieving great things.

**Reply by Angamanga** [reply]
Welcome @Mekebib-Ergudo! 🎉

**Comment by mhdamaan79** [comment]
👋 Hello everyone! I'm Amaan, an Outreachy applicant for May 2024. Excited to dive into Ushahidi's contribution journey!
![Screenshot 2024-03-05 144318](https://github.com/ushahidi/platform/assets/118375524/425df67e-5236-4364-bcfa-617f80945984)
![dockerushahida](https://github.com/ushahidi/platform/assets/118375524/352764c7-adc2-4a9a-92dd-f9d5b0594919)

**Reply by Angamanga** [reply]
Welcome @mhdamaan79! 🎉

**Reply by kofodym** [reply]
Welcome. It's nice to have you here

**Comment by rimjhimittal** [comment]
Hello everyone! I'm Rimjhim, an Outreachy applicant for 2024, eagerly looking forward to contributing to Ushahidi and making a positive impact together!

**Reply by Angamanga** [reply]
Welcome @rimjhimittal! 🎉

**Reply by kofodym** [reply]
Welcome

**Comment by Felysilver** [comment]
Hello everyone! My name is Fely an Outreachy applicant. My system is up and running. So I am ready to take on some issues and contribute with the help of the community. I have set up he [Ushahidi Platform API](https://github.com/ushahidi/platform) as illustrated here
![api](https://github.com/ushahidi/platform/assets/162261453/87f62299-df6f-4c30-9fa6-1794037fdc7e)
then set up the platform-client for development as here Thanks!

**Reply by kofodym** [reply]
Wow. Great. I might need so help as I an currently setting mine up.

**Reply by Angamanga** [reply]
Welcome @Felysilver! 🎉

**Comment by sanketmp** [comment]
Hi everyone! I'm Sanket, an Outreachy applicant for May 2024. I am excited to contribute to Ushahidi. Cheers!!

**Reply by kofodym** [reply]
Welcome. Nice to have you here

**Reply by Angamanga** [reply]
Welcome @sanketmp! 🎉

**Comment by NonsoBarn** [comment]
hello! My name is Nonso, I am an Outreachy applicant for 2024. I am excited to be here and also to learn contribute to the Ushahidi project.

**Reply by kofodym** [reply]
Welcome Nonso

**Reply by Angamanga** [reply]
Welcome @NonsoBarn! 🎉

**Comment by Molatii** [comment]
Hello everyone! I'm Paballo Molati, an Outreachy May 2024 applicant from the beautiful kingdom of Lesotho. I am thrilled about the prospect of contributing to Ushahidi and working together to make a positive impact!

**Reply by Angamanga** [reply]
Welcome @Molatii! 🎉

**Reply by Molatii** [reply]
Thank you..

**Comment by Bilalshaikh15** [comment]
Hello @Angamanga i am bilal shaikh an outreach intern I have successfully setup developer-environment on my windows.
I am eager and thrilled to contribute to the project.
just wanted to ask that should i wait for the issue to get assigned it to me or should i start working immediately even if someone else is working on it?
![Screenshot 2024-03-05 193906](https://github.com/ushahidi/platform/assets/86535450/92850f93-ec01-4d71-b072-9e50c63607e7)

**Reply by Angamanga** [reply]
Welcome @Bilalshaikh15! Go through the unassigned issues in this list and ping me on one that sparks your interest! https://github.com/orgs/ushahidi/projects/18/views/1?sliceBy%5Bvalue%5D=Outreachy%3A+Accessibility+2024

**Comment by MugoBrian** [comment]
Hello everyone.My name is Brian Mugo from Kenya, I am eager to contribute to the Ushahidi Web accessibility project as an Outreachy applicant. Looking forward to learning and working with everyone. Cheers :)

**Reply by Angamanga** [reply]
Hi @MugoBrian! Welcome :)

**Reply by MugoBrian** [reply]
Thank you.

**Comment by MugoBrian** [comment]
My setup is up and running. 
![Screenshot from 2024-03-05 17-28-01](https://github.com/ushahidi/platform/assets/49617104/99d4a2ab-2cee-4720-af24-b12699dd647e)

**Reply by Angamanga** [reply]
Well done @MugoBrian 🎉

**Comment by Chiemezuo** [comment]
Hello!!!
I'm Chiemezuo ('Mezuo 'for short) from Nigeria. I'm glad to be here, and I look forward to all the learning and fun that comes with open-source collaboration. 😁 

Here's a picture of my setup running: 

![Screenshot 2024-03-05 133103](https://github.com/ushahidi/platform/assets/29470516/ce4b94a5-2feb-4710-8b70-03c2404cbb89)

>**Side Note**: In the setup docs [here](https://docs.ushahidi.com/platform-developer-documentation/v/set-up-the-developer-environment-for-ushahidi/set-up-your-developer-environment/setting-up-the-platform-client-for-development), there's a small issue. At number 6, the default password isn't `admin`, it's `admin123`.

**Reply by MugoBrian** [reply]
Welcome. Great to have you, I am looking forward to working with you again.

**Reply by Chiemezuo** [reply]
Honestly, it's good to see you again buddy. 😁
Looking forward to a collab.

**Reply by Angamanga** [reply]
@Chiemezuo Welcome to Ushahidi! 🎉

**Reply by Chiemezuo** [reply]
Thank you!!! @Angamanga 😁

**Comment by ademolaomosanya** [comment]
Hello Ademola,
so glad to be here i have the platform-client-mzima repo running on my localhost 
hoping to make  great contibutions to the great project
<img width="1280" alt="Screenshot 2024-03-05 at 20 59 01" src="https://github.com/ushahidi/platform/assets/87182204/0a944dba-8d98-4c7e-af6b-885dff2a19b1">

**Comment by pamisijohn** [comment]
Hi!! I'm John, very excited to be here I hope to learn and give quality contribution!

**Comment by RWUBAKWANAYO** [comment]
Hey everyone, I'm Olivier. I'm happy to be here and excited to share and learn from all of you.

**Comment by kevincodesa** [comment]
Hello everyone, am Nalutaaya Kevin from Uganda. Am so excited to be part of Outreachy 2024 and willing to contribute with everyone for the better good. Thanks

**Comment by PreciousAma** [comment]
Hi @Angamanga, I'm Chilaka Precious a current outreachy applicant,  I'm trying to join the Ushahidi community on discord but I keep getting an error message "Invalid Invite". how do I get this fixed so I can start contributing?

**Reply by PreciousAma** [reply]
i have been able to join the discord channel, thank you.

**Reply by JO-YE** [reply]
hello Precious, How were you able to join? Am getting the same error message here

**Reply by PreciousAma** [reply]
Hi, I had to create a new discord account(with a new email and phone number)  as other options proved abortive.

**Reply by JO-YE** [reply]
ohh wow. will try doing that. Thanks

**Comment by Oluwakunmi-Obaditan** [comment]
Hi Everyone, I'm Kunmi and I'm excited to contribute to this project in the Ushahidi community. Thank you.

**Comment by JO-YE** [comment]
Hello!

I'm Racheal from Nigeria, and I'm thrilled to join the family. I'm particularly excited about collaborating on the Accessibility review and improvement of the Ushahidi Platform Client. Looking forward to working together and making a positive impact!

**Comment by JO-YE** [comment]
I have not been able to join the Discord community. Getting 'unable to accept invite' message. Can anyone please help out with that.

**Reply by Oluwakunmi-Obaditan** [reply]
Try using another email, it would allow you to. When you're done, go back and copy the link to the "Community Discord server" and paste it into the search bar, you will be able to join the community that way.

**Reply by JO-YE** [reply]
Got it sorted out. Thank you.

I logged in to my discord account on my browser in the same window, then re-clicked on the link. that took me directly to the channel.

**Comment by prathamkandari** [comment]
Hi everyone, I'm Pratham Kandari, an applicant for Outreachy 2024. I'm eager to collaborate on the project and tackle its challenges.

**Comment by xfy0012** [comment]
hello everyone I am fanying xu, an applicant for Outreachy 2024. I am excited to be a contributor on the project!

**Comment by Dorcas-BD** [comment]
Hello everyone, I'm sooo glad to dive into Ushahidi Platform...

![Ushahidi Api](https://github.com/ushahidi/platform/assets/52034693/9c04baf5-f697-4e66-9f24-cbfbd8b8135f)
![Ushahidi Client](https://github.com/ushahidi/platform/assets/52034693/f8140799-7818-41a3-a009-8edd5bcfadab)
![Ushahidi Terminal](https://github.com/ushahidi/platform/assets/52034693/5aa8b5d5-f549-4da0-8f0e-06826a66844b)

**Comment by RashJrEdmund** [comment]
Hello everyone, I'm Lahfen Orashus, Rash for short, an Outreachy applicant from Cameroon. Yesterday, I got my setup up, and running successfully. Still going through the site in search for issues to report, 

![Screenshot from 2024-03-10 09-36-39](https://github.com/ushahidi/platform/assets/116172031/7f4b34c8-b493-4fce-8bac-3ba7d4ff14f4)

![Screenshot from 2024-03-10 09-37-33](https://github.com/ushahidi/platform/assets/116172031/2f59269c-c759-4e2b-828a-eaf4a9983008)

![Screenshot from 2024-03-10 09-37-58](https://github.com/ushahidi/platform/assets/116172031/4b8cad50-2093-49e6-9f53-7b0343807b46)

**Comment by Mekebib-Ergudo** [comment]
![Screenshot 2024-03-11 at 11 58 23 AM](https://github.com/ushahidi/platform/assets/134989328/d6ab3bfc-dc94-48bb-8958-9abd5d0caa13)
<img width="1512" alt="Screenshot 2024-03-11 at 11 58 38 AM" src="https://github.com/ushahidi/platform/assets/134989328/541d2718-768f-481f-8e0f-83fe3cbb86e9">

**Reply by Mekebib-Ergudo** [reply]
Just set up the Dev. environment and ready to take the Challenge !!

**Comment by Jipheens** [comment]
Hello everyone.My name is Jipheens Wahome from Kenya, I am so exited to 
![1](https://github.com/ushahidi/platform/assets/17562987/77257559-86c4-4507-8172-9b39530c6706)
contribute to the Ushahidi Web accessibility project as an Outreachy applicant. Looking forward to learning and working with everyone. 
![2](https://github.com/ushahidi/platform/assets/17562987/f5fdc6a7-929a-4f9e-ae45-a9a66805c7fb)
![1](https://github.com/ushahidi/platform/assets/17562987/769b45c7-6067-4a1c-9671-c5d481c38e06)

**Comment by lillian-busingye** [comment]
<img width="1664" alt="Screenshot 2024-03-12 at 10 38 01" src="https://github.com/ushahidi/platform/assets/143929780/526486d8-13f5-4ed0-9983-3638016f4f12">
Done with my set up.

**Comment by ssemanich** [comment]
Hello everyone.This is Semango Nicholas.Am a new intern at ushahid.am happy to be here

**Reply by blessdamy** [reply]
Welcome @ssemanich
